### PR TITLE
[LM-10858] Community Configurations

### PR DESCRIPTION
**Pull request by sidcr7-likeminds**
## Description

**Features**

- Added community configurations call
- Added post as variable

**Bugs**

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Hotfix (fix for breaking changes)
- [x] New Feature (non-breaking change which adds functionality)
- [ ] Other, please describe:

### Author Checklist

- [x] Assignee added
- [x] Reviewer added
- [x] Description added
- [ ] Unit test cases added

### Reviewer Checklist

- [x] PR title starts with Jira ticket number prefix
- [ ] If hotfix, Add Hotfix as prefix in PR title
- [x] Label added
- [x] Commit messages follows standard format (added, changed, removed, code commented and chore)
- [x] Description about the Feature/ Bugs added

### CODE REVIEW

File: likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/activityfeed/view/LMFeedActivityFeedFragment.kt [hunk 1]
```diff
@@ -74,6 +95,18 @@ open class LMFeedActivityFeedFragment : Fragment(), LMFeedActivityFeedAdapterLis
...
 
+    private fun initListeners() {
+        binding.apply {
+            headerViewActivityFeed.setNavigationIconClickListener {
+                onNavigationIconClick()
+            }
+        }
+    }
+
+    protected open fun onNavigationIconClick() {
```
Comment (ishaan1607) [inline likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/activityfeed/view/LMFeedActivityFeedFragment.kt:106]: add comments

File: likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/post/edit/view/LMFeedEditPostDisabledTopicsDialogFragment.kt [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        binding = LmFeedDialogFragmentEditPostDisabledTopicsBinding.inflate(layoutInflater)
+
+        binding.apply {
+            customizeDisabledTopicsDialog(alertDialogDisabledTopics)
+            return root
```
Comment (ishaan1607) [inline likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/post/edit/view/LMFeedEditPostDisabledTopicsDialogFragment.kt:60]: return should be out from `apply` block

File: likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/ui/base/styles/LMFeedChipStyle.kt [hunk 3]
```diff
@@ -31,7 +30,10 @@ class LMFeedChipStyle private constructor(
     @DimenRes val chipStartPadding: Int?,
     @DimenRes val chipEndPadding: Int?,
     @DimenRes val chipCornerRadius: Int,
-    @DimenRes val chipTextSize: Int
+    @DimenRes val chipTextSize: Int,
+    @DrawableRes val chipIcon: Int?,
+    @DimenRes val chipIconSize: Int?,
+    @ColorRes val chipIconTint: Int?
```
Comment (ishaan1607) [inline likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/ui/base/styles/LMFeedChipStyle.kt:36]: add these changes in styling comments

File: likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/ui/widgets/comment/commentcomposer/style/LMFeedCommentComposerViewStyle.kt [hunk 4]
```diff
@@ -25,9 +27,10 @@ class LMFeedCommentComposerViewStyle private constructor(
     //replying to view style
     val replyingToStyle: LMFeedTextStyle?,
     //remove replying to view style
-    val removeReplyingToStyle: LMFeedIconStyle?
+    val removeReplyingToStyle: LMFeedIconStyle?,
+    @DimenRes val elevation: Int?,
+    @ColorRes val backgroundColor: Int?
```
Comment (ishaan1607) [inline likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/ui/widgets/comment/commentcomposer/style/LMFeedCommentComposerViewStyle.kt:32]: add these changes in styling comments

File: likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/ui/widgets/headerview/style/LMFeedHeaderViewStyle.kt [hunk 5]
```diff
@@ -31,12 +30,18 @@ class LMFeedHeaderViewStyle private constructor(
...
     val navigationIconStyle: LMFeedIconStyle?,
     //search icon style
-    val searchIconStyle: LMFeedIconStyle?
+    val searchIconStyle: LMFeedIconStyle?,
+    //profile image style in header
+    val userProfileStyle: LMFeedImageStyle?,
+    //notification icon style
+    val notificationIconStyle: LMFeedIconStyle?,
+    //notification count style
+    val notificationCountTextStyle: LMFeedTextStyle?,
```
Comment (ishaan1607) [inline likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/ui/widgets/headerview/style/LMFeedHeaderViewStyle.kt:44]: add these changes in styling comments

File: likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/ui/widgets/headerview/view/LMFeedHeaderView.kt [hunk 6]
```diff
@@ -172,6 +212,71 @@ class LMFeedHeaderView : ConstraintLayout {
         }
     }
 
+    fun setNotificationCountText(count: Int) {
```
Comment (ishaan1607) [inline likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/ui/widgets/headerview/view/LMFeedHeaderView.kt:215]: add comments

File: likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/ui/widgets/headerview/view/LMFeedHeaderView.kt [hunk 7]
```diff
@@ -172,6 +212,71 @@ class LMFeedHeaderView : ConstraintLayout {
...
+                }
+
+                else -> {
+                    configureNotificationBadge(context.getString(R.string.lm_feed_nine_nine_plus))
+                }
+            }
+        }
+    }
+
+    fun setNotificationIconVisibility(isVisible: Boolean) {
```
Comment (ishaan1607) [inline likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/ui/widgets/headerview/view/LMFeedHeaderView.kt:235]: add comments

File: likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/ui/widgets/noentitylayout/view/LMFeedNoEntityLayoutView.kt [hunk 8]
```diff
@@ -66,6 +62,11 @@ class LMFeedNoEntityLayoutView : ConstraintLayout {
         binding.fabAction.text = ctaAction
     }
 
+    fun setActionFABColor(@ColorRes fabColor: Int) {
```
Comment (ishaan1607) [inline likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/ui/widgets/noentitylayout/view/LMFeedNoEntityLayoutView.kt:65]: add comments

File: likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/universalfeed/view/LMFeedUniversalFeedFragment.kt [hunk 9]
```diff
@@ -647,13 +741,90 @@ open class LMFeedUniversalFeedFragment :
...
+            text = getString(
+                R.string.lm_feed_new_s,
+                LMFeedCommunityUtil.getPostVariable()
+                    .pluralizeOrCapitalize(LMFeedWordAction.ALL_CAPITAL_SINGULAR)
+            )
         }
     }
 
-    protected open fun onCreateNewPostClick() {
+    protected open fun onCreateNewPostClick(hasCreatePostRights: Boolean) {
```
Comment (ishaan1607) [inline likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/universalfeed/view/LMFeedUniversalFeedFragment.kt:776]: add commeents

File: likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/utils/user/LMFeedConnectUser.kt [hunk 10]
```diff
@@ -19,6 +22,10 @@ class LMFeedConnectUser private constructor(
     val deviceId: String,
     val enablePushNotifications: Boolean
 ) {
+    companion object {
+
+    }
```
Comment (ishaan1607) [inline likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/utils/user/LMFeedConnectUser.kt:27]: empty block?

**Review by ishaan1607 — CHANGES_REQUESTED** [review]

File: likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/post/edit/view/LMFeedEditPostDisabledTopicsDialogFragment.kt (hunk 2 above)
Comment (sidcr7-likeminds) [reply likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/post/edit/view/LMFeedEditPostDisabledTopicsDialogFragment.kt:60]: why?

**Review by ishaan1607 — APPROVED** [review]

**Pull request merged**